torch_glow/tests/nodes/quantized_linear_test.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import torch
+
+from tests.utils import jitVsGlow
+import torch_glow
+import pytest
+
+@pytest.mark.skip("not ready")
+def test_quantized_linear():
+    """Basic test of the PyTorch quantized::linear Node on Glow."""
+
+    def test_f(inputs, weights, bias=None):
+        q_int = torch.nn.quantized.Quantize(1.0/64, 2, torch.qint8)
+        q_uint = torch.nn.quantized.Quantize(1.0, 0, torch.quint8)
+
+        dq = torch.nn.quantized.DeQuantize()
+
+        q_inputs = q_uint(inputs)
+        q_weights = q_int(weights)
+
+        return dq(torch.nn.quantized.functional.linear(q_inputs, q_weights, bias))
+
+    n = 5
+    in_features = 4
+    out_features = 3
+
+    inputs = torch.randn(n, in_features)
+    weights = torch.randn(out_features, in_features)
+
+    jitVsGlow(test_f, inputs, weights, expected_fused_ops={"glow::unpacked_quantized_linear", "aten::quantize_per_tensor", "aten::dequantize"})
+
+def test_quantized_linear_with_bias():
+    """Basic test of the PyTorch quantized::linear Node on Glow."""
+
+    def test_f(inputs, weights, bias=None):
+        q_int = torch.nn.quantized.Quantize(1.0/64, 2, torch.qint8)
+        q_uint = torch.nn.quantized.Quantize(1.0, 0, torch.quint8)
+
+        dq = torch.nn.quantized.DeQuantize()
+
+        q_inputs = q_uint(inputs)
+        q_weights = q_int(weights)
+
+        return dq(torch.nn.quantized.functional.linear(q_inputs, q_weights, bias))
```

Comment:
If we don't add `dq` here, what is going to happen?

Reply:
It won't work because `elemKindToScalarType()` doesn't know how to translate glow quantized types to pytorch quantized types yet but it should be pretty easy to add

Reply:
Yeah, I think one the things is to look into this "intermediate" blobs. @zrphercule, are yo covering this?